packages/manager/modules/pci/src/projects/project/inactive/inactive.html

```diff
@@ -0,0 +1,43 @@
+<div class="oui-modal oui-modal_shadow">
```

Comment:
`oui-modal` component is not suiting the needs ?

Reply:
Replaced with `oui-modal`. Thanks for the suggestion